assets/Product.wxs

```diff
@@ -10,14 +10,16 @@
   <?define ProductName = "$(env.ProductName)" ?>
   <?define ProductGuid = "$(env.ProductGuid)" ?>
   <?define ProductVersion = "$(env.ProductVersion)" ?>
+  <?define ProductSemanticVersion = "$(env.ProductSemanticVersion)" ?>
   <?define ProductVersionWithName = "$(var.ProductName)_$(var.ProductVersion)"?>
+  <?define ProductSemanticVersionWithName = "$(var.ProductName)_$(env.ProductSemanticVersion)"?>
```

Comment:
I will be standardizing on dashes for package names also.
example:
PowerShell-6.0.0-alpha.17-win10-win2016-x64.msi
PowerShell-6.0.0-alpha.17-win10-win2016-x64.zip